### PR TITLE
Update example placeholder for labels section of odc add flows

### PR DESCRIPTION
**Pull request by divyanshiGupta**
**Fixes:** https://issues.redhat.com/browse/ODC-6027

This PR updates example placeholder for labels section as `app` label can be misleading for the users since it is a default label and hence is added always when an app is created using add flows and also since only `user-labels` are editable via edit flows.

**Screenshot:**
Screenshot 2021-06-16 at 5 54 22 PM

cc: @openshift/team-devconsole-ux 

/kind bug

### CODE REVIEW

**Comment by divyanshiGupta** [comment]
@beaumorley PTAL at this PR. I have updated the label placeholder to be `app.io/type=frontend` if that looks fine to you can add your approval.

**Review by rohitkrai03 — APPROVED** [review]
/approve

Code changes looks good to me. Will wait for UX to approve the placeholder text itself.

**Comment by beaumorley** [comment]
I am not an expert in the syntax. If the syntax is correct, from UX POV it looks good. Thanks.

**Comment by nemesis09** [comment]
verified locally
has approval from UX for the changes.
/lgtm

**Pull request merged**
